### PR TITLE
Update logging.rst

### PR DESCRIPTION
**Pull request by SychevIgor**
Links to source code on a GitHub updated to current version rc1.
And fix for pull request https://github.com/aspnet/Docs/pull/695/

### CODE REVIEW

**Comment by danroth27** [comment]
:shipit:

File: aspnet/fundamentals/logging.rst [hunk 1]
```diff
@@ -119,7 +119,7 @@ The console logger prefixes verbose output with "verbose: " and uses a gray font
...
+In the course of logging information within your application, you can group a set of logical operations within a *scope*. A scope is an ``IDisposable`` type returned by calling the ``BeginScopeImpl`` method, which lasts from the moment it is created until it is disposed. Not all loggers support scopes. For example, the `ConsoleLogger simply returns null <https://github.com/aspnet/Logging/blob/1.0.0-rc1/src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs#L114>`_ from ``BeginScopeImpl``. The built-in `TraceSource logger <https://github.com/aspnet/Logging/blob/1.0.0-rc1/src/Microsoft.Extensions.Logging.TraceSource/TraceSourceLogger.cs#L66-L69>`_ returns a scope instance that is responsible for starting and stopping tracing operations. Any logging state, such as a transaction id, is attached to the scope when it is created.
```
Comment (shirhatti) [inline aspnet/fundamentals/logging.rst]: ~~For example, the `ConsoleLogger simply returns null~~

This is no longer true in rc1. Could you drop that?

**Comment by shirhatti** [comment]
Merged in ff52b01882ebb6c5b74c2412b6a0faf8ac17edbf

File: aspnet/fundamentals/logging.rst (hunk 1 above)
Comment (shirhatti) [inline aspnet/fundamentals/logging.rst]: Fixed in 0ca4ed005acec6fa1729312abf3ed2390f577c60

**Pull request closed without merging**